### ISSUE TITLE
Add a godot-ideas referral to the issue template

### ISSUE THREAD

**Issue by willnationsdev** [godotengine/godot-proposals#91]
**Describe the project you are working on:**
Managing community expectations for the Godot-Proposals repository.

**Describe how this feature / enhancement will help your project:**
I think we should add a blurb in the `godot-proposals` issue template that redirects people to the new `godot-extended-libraries/godot-ideas` repository if they do not feel as though they can adequately answer the questions, or if they already know that their proposal is not applicable to the engine development itself.

The reason this would help, aside from things already brought up in #47, is that it drives users awareness of this space up.

**Show a mock up screenshots/video or a flow diagram explaining how your proposal will work:**

See below.

**Describe implementation detail for your proposal (in code), if possible:**

Just add a comment to the top of the message, exactly like the one in `godotengine/godot` that tells people they can go to `godot-extended-libraries/godot-ideas` instead.

> \<!--If you feel that you cannot provide highly detailed instructions with the proposal below, consider creating a more simple, open-ended Issue in the unofficial, community-maintained [godot-extended-libraries/godot-ideas repository](https://github.com/godot-extended-libraries/godot-ideas/issues/new)--\>

Something like that.

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**

Relates to the GitHub repository, not Godot itself.

**Is there a reason why this should be core and not an add-on in the asset library?:**

See above.

**Comment by willnationsdev** [comment]
cc @Xrayez @clayjohn

**Comment by Xrayez** [comment]
> See below.
> See above.

I guess no comments needed. 🙂

But yeah there are stuff which doesn't fit `godot-proposals`, like extended discussions, trackers, controversial topics etc. Also, this can channel proposals like #58 to community place.

**Comment by golddotasksquestions** [comment]
Unfortunately as it is right now, I feel like the reddit post alone adds more confusion than anything.
For this to work and be of actual value, instructions and purpose have to be crystal clear and super easy grasp to a user who never participated or cared about this discussion.
Channeling also needs to happen at the Point Of Interest. Which is the place where people go to submit an issue. 
As someone has mentioned in the reddit post, it also needs official blessing. So if core devs want this to work, there will have to be a blogpost about this as well to make clear this is not a legitimate way to voice ideas concerns, brainstorm or have a discussion about plugins and modules.

**Comment by willnationsdev** [comment]
@golddotasksquestions 

Hard to argue with the reality of things. No official support means it's more likely to just die. I'll maintain it either way, since I think it has a valid use-case, but we'll just have to wait and see what akien/reduz decide to do.

**Comment by Xrayez** [comment]
Any updates? Almost a year passed. 😕

@akien-mga we'd like to know your opinion regarding this proposal. Look at the sheer number of current proposals which won't be ever implemented in Godot directly (as previously stated by clayjohn), some are just discussions. But if that's acceptable for GIP, than that's totally fine as-is, but then we'd like to expect an appropriate conduct from the core developers, currently they outright express disapproval for something which does not belong to Godot itself (at least the way I see it, lately I've noticed that some are just getting tired to repeat the same song of "this should be a plugin" 🙂). And probably rightly so, because the scope of proposed features should also be defined, currently people assume that Godot is still a small engine which keeps growing, but according to the development direction I've experienced myself, it seems like it's not the case, the current direction is exactly to keep the engine *as small as possible at its core*, so we just ask for clarifications and coming up with correct expectations, I see nothing wrong with that personally.
